ibis/expr/operations.py

```diff
@@ -553,7 +553,14 @@ class FuzzySearch(BooleanValueOp):
 
 
 class StringSQLLike(FuzzySearch):
-    pass
+
+    input_type = [
+        string,
+        string(name='pattern'),
+        rules.instance_of(
+            six.string_types, name='escape', default=None, optional=True
+        ),
```

Comment:
I introduced an `escape` argument to allow users to escape special patterns like `%` and `_`.